docs/pycharm_debugging.md

```diff
@@ -1,47 +1,47 @@
-Debugging with PyCharm

```

Comment:
This file looks the same to me, why is github showing it as changed?

Reply:
Can you revert this file, assuming you didn't actually make changes to it? You should be able to do `git restore -s dev docs/pycharm_debugging.md` to revert it, and then  if you add and commit it, this should disappear.